services/hanke-service/src/main/kotlin/fi/hel/haitaton/hanke/permissions/HankeKayttajaService.kt

```diff
@@ -511,6 +538,15 @@ class HankeKayttajaService(
         hankekayttajaRepository.findByHankeIdAndSahkopostiIn(hankeId, emails).map { it.sahkoposti }
 }
 
+data class OwnContactUpdate(@NotBlank val sahkoposti: String, @NotBlank val puhelinnumero: String)
+
+data class ContactUpdate(
```

Comment:
I don't think names can be counted as contact information. This could be e.g. KayttajaUpdate instead.